apps/aestratum/src/aestratum.app.src

```diff
@@ -0,0 +1,19 @@
+{application, aestratum,
+ [{description, "AE Stratum"},
+  {vsn, "0.1.0"},
```

Comment:
```suggestion
  {vsn, {cmd, "cat ../../VERSION"}},
```

Idea is reducing ambiguity in discerning node version from logs submitted by user.